b2b_ecommerce/models.py

```diff
@@ -32,9 +33,13 @@ class B2BOrder(OrderAbstract, AuditableModel):
         CouponPaymentVersion, null=True, on_delete=models.PROTECT
     )
 
-    reference_number_prefix = REFERENCE_NUMBER_PREFIX
     objects = OrderManager()
 
+    @staticmethod
+    def get_reference_number_prefix():
```

Comment:
The function name needs to be the same for both because `OrderManager.get_by_reference_number` uses `self.model` which could be either `Order` or `B2BOrder`.